### PR TITLE
[MB-16191] create table and model for service request documents and uploads

### PR DESCRIPTION
**Pull request by pearl-truss**
## Summary

This PR creates 2 new database tables and their corresponding models

Note: In addition to the service request documents, I also created the table necessary for storing the documents associated uploads that isn't tied to proof of service docs. 

These changes were modeled after `proof_of_service_docs` and `prime_uploads` tables

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [x] Have the Jira acceptance criteria been met for this change?

#### Any new migrations/schema changes:

- [x] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [x] Have been communicated to #g-database.
- [x] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

### CODE REVIEW

File: migrations/app/schema/20230602140622_creating_service_request_uploads_table.up.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
+create table service_request_document_uploads
+(
+    id uuid not null
```
Comment (kctruss) [inline migrations/app/schema/20230602140622_creating_service_request_uploads_table.up.sql:3]: **nitpick:** `not null` is not necessary for primary key fields, and (at least in my DB tool) doesn't end up carrying through to the table structure‏

File: migrations/app/schema/20230602140622_creating_service_request_uploads_table.up.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+            references service_request_documents not null,
+    contractor_id uuid not null constraint service_request_documents_contractor_id_fkey references contractors,
+    upload_id uuid not null constraint service_request_documents_uploads_id_fkey references uploads,
+    created_at timestamp not null,
+    updated_at timestamp not null,
+    deleted_at timestamp with time zone
+
+);
+
+-- Column Comments
```
Comment (kctruss) [inline migrations/app/schema/20230602140622_creating_service_request_uploads_table.up.sql:16]: **suggestion (non-blocking):** Could you add comments for `contractor_id` and `service_request_documents_id` as well? ‏

**Review by kctruss — APPROVED** [review]
Migrations look good! Just added two comments that I wouldn't consider blockers.

**Comment by pearl-truss** [comment]
> Migrations look good! Just added two comments that I wouldn't consider blockers.

Thanks @kctruss for the prompt review! I've pushed changes to address both comments

**Review by YanZ777 — APPROVED** [review]

**Pull request merged**